OpenSim/Common/C3DFileAdapter.cpp

```diff
@@ -42,7 +42,7 @@ C3DFileAdapter*
 C3DFileAdapter::clone() const {
     return new C3DFileAdapter{*this};
 }
-
+#if 0
```

Comment:
Looks like the PR is in good enough shape that all the code with `#if 0 ...` could be removed at this point.

Reply:
Done